Tutorials/G-CNN_Honeycomb.ipynb

```diff
@@ -0,0 +1,812 @@
+{
```

Comment:
<p>"Marhall-Perels sign rule" -&gt; "Marshall-Peierls sign rule"</p><p> </p><p>Just out of curiosity: Does the GCNN have difficulties learning the signs if we set <code>sign_rule=False</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/647/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I don’t the G-CNN would have trouble learning a simple sign rule like this (I haven’t tried yet) but it would definitely struggle with more difficult lattices if you optimized the amplitudes+phases together from the start (this is a pretty universal problem).</p><p> </p><p>Usually it’s good practice to optimize the phases first (I.e. return 1j*Im(output)) from the wave function) and then train the amplitudes. With that practice, I’ve gotten G-CNN to learn the signs of Pyrochlore, Kagome, etc., with a little work of picking a good learning rate</p><p> </p>

Reply:
Maybe write this down? it's interesting discussion

Reply:
Yeah I added a comment. It’d be nice to have a separate tutorial. It’s just hard to find a problem that is difficult enough to need phase optimization and easy enough for a tutorial

Reply:
Thanks! I know how hard is to write good tutorials, but I think some of those issues are worth just 'throwing' them there for the interested reader. If people care, they'll notice that someone thought about it, else, they'll just ignore it.

Reply:
Agreed!